perfkitbenchmarker/linux_packages/mongodb_server.py

```diff
@@ -18,6 +18,7 @@
 
 def YumInstall(vm):
   """Installs the mongodb package on the VM."""
+  vm.RemoteCommand('sudo setenforce 0')
```

Comment:
Sounds good and thanks for looking.

LGTM

I'll complete the pull request this morning.